testing/test_notebooks.py

```diff
@@ -32,6 +32,7 @@
     'kernel_shap_adult_lr.ipynb',  # slow to explain 128 instances
     'xgboost_model_fitting_adult.ipynb',  # very expensive hyperparameter tuning
     'integrated_gradients_transformers.ipynb',  # forward pass through BERT to get embeddings is very slow
+    'protoselect_adult_cifar10.ipynb',  # wget cifar10 model from google cloud
```

Comment:
Why is this a reason not to test? Fetching the model should be quick enough.

Reply:
It's not. Forgot to remove it after I tested something. Some tests were failing and complaining about `wget`.

Reply:
To be excluded on Windows due to use of `wget`.